test/pt2_to_qcircle_test/builder.py

```diff
@@ -25,6 +25,8 @@
 
 from test.utils.base_builders import TestDictBuilderBase, TestRunnerBase
 
+IS_CI_MODE = os.environ.get("RUN_INTERNAL_TESTS", "0") == "1"
```

Comment:
```suggestion
CI_MODE = os.environ.get("RUN_INTERNAL_TESTS", "0") == "1"
```
Or it can be an another candidate

Reply:
All candidates look good. Let me go as it is:)